corehq/apps/app_manager/templates/app_manager/partials/modules/style_configuration_modal.html

```diff
@@ -20,7 +20,7 @@ <h4 class="modal-title">{% trans "Edit Field Style" %}</h4>
               </select>
             </div>
             <div class="form-group">
-              <label for="verticalAlign" class="control-label">{% trans "Vertial Align" %}</label>
+              <label for="verticalAlign" class="control-label">{% trans "Vertical Align" %}</label>
```

Comment:
Is it worth checking the po files if there are already any translations and updating the keys?

Reply:
Just checked. They all map to "". But it might still be worthwhile to replacing them to keep the po files clean.

Reply:
Thank you for checking. Given that the po files are automatically generated, how does me updating them benefit anything?

Reply:
I thought the old "Vertial Align" might be sticking around. But I guess it does not.